app/src/main/java/com/example/tungcon/demo/movie/MovieAdapter.java

```diff
@@ -0,0 +1,94 @@
+package com.example.tungcon.demo.movie;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.CardView;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.RatingBar;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.example.tungcon.demo.R;
+import com.example.tungcon.demo.data.model.Movie;
+
+
+import java.util.List;
+
+public class MovieAdapter extends RecyclerView.Adapter<MovieAdapter.ViewHolder> {
+    private Context mContext;
+    private List<Movie> mMovieList;
+    OnImageClickListener mOnImageClickListener;
+
+    public MovieAdapter(Context mContext, List<Movie> mMovieList,
+                        OnImageClickListener mOnImageClickListener) {
+        this.mContext = mContext;
+        this.mMovieList = mMovieList;
+        this.mOnImageClickListener = mOnImageClickListener;
+    }
+
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        LayoutInflater inflater = LayoutInflater.from(mContext);
+        View view = inflater.inflate(R.layout.item_movie, parent, false);
+        return new ViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
+        holder.bindData(mMovieList.get(position));
+    }
+
+    @Override
+    public int getItemCount() {
+        return mMovieList.size();
```

Comment:
check null  mMovieList if mMovieList ==null app crash 
```suggestion
        return mMovieList!=null? mMovieList.size(): 0;
```